src/services/contentAggregator.js

```diff
@@ -17,33 +22,52 @@ export async function addContextToContent(dataPromise, ...dataArgs)
     addProgressStatus = false,
     addResumeTimeSeconds = false,
     addLastInteractedChild = false,
+    addNextLesson = false,
+    lastInteractedParent = false,
```

Comment:
```suggestion
    addLastInteractedParent = false,
```
to match param name format